src/components/MultiGestureCanvas/types.ts

```diff
@@ -20,7 +20,10 @@ type ZoomRange = {
 };
 
 /** Triggered whenever the scale of the MultiGestureCanvas changes */
-type OnScaleChangedCallback = (zoomScale: number) => void;
+type OnScaleChangedCallback = ((zoomScale: number) => void) | undefined;
```

Comment:
```suggestion
type OnScaleChangedCallback = ((zoomScale: number) => void);
```

We don't need to union with `undefined` here. If the callback is not always required just make the prop that uses it optional